<launch>
    <arg name="semantic_tf" default="true"/>


    <group if="$semantic_tf">
        <node pkg="tf" type="static_transform_publisher" name="map_align" args="0 0 0 -0.15 0 0 map map_align 100" />
    </group>

    <rosparam command="load" file="$(find robot_map)/config/semantic_map.yaml" />

    <node pkg="robot_map" type="semantic_map.py" name="semantic_map" args="map_align" output="screen">
        <!--<remap from="/camera_info" to="/stereo/$(arg eye)/camera_info" />-->
        <!--<param name="image_is_rectified" value="True"/>-->

    </node>






</launch>
